### PR TITLE
alluvial fans update

### PR DESCRIPTION
**Pull request by rkelson**
add links to status graphics
add link to study
add link to WIP data

### CODE REVIEW

File: data/geoscience/alluvial-fans/index.html [hunk 1]
```diff
@@ -37,6 +39,9 @@ <h3 id="AlluvialFans">
     </h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
+    <a alt="Alluvial Fan Mapping Status" href="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019_sm.jpg" | prepend: site.baseurl }}">
```
Comment (stdavis) [inline data/geoscience/alluvial-fans/index.html]:
```suggestion
    <a alt="Alluvial Fan Mapping Status" href="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019_sm.png" | prepend: site.baseurl }}">
```

File: data/geoscience/alluvial-fans/index.html [hunk 2]
```diff
@@ -37,6 +39,9 @@ <h3 id="AlluvialFans">
...
+      <img class="inline-text-left outline" src="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019.jpg" | prepend: site.baseurl }}" alt="Alluvial Fan Mapping Status" />
```
Comment (stdavis) [inline data/geoscience/alluvial-fans/index.html]:
```suggestion
      <img class="inline-text-left outline" src="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019.png" | prepend: site.baseurl }}" alt="Alluvial Fan Mapping Status" />
```

File: data/geoscience/alluvial-fans/index.html [hunk 3]
```diff
@@ -37,6 +39,9 @@ <h3 id="AlluvialFans">
     </h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
+    <a alt="Alluvial Fan Mapping Status" href="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019_sm.png" | prepend: site.baseurl }}">
```
Comment (stdavis) [inline data/geoscience/alluvial-fans/index.html]:
```suggestion
    <a alt="Alluvial Fan Mapping Status" href="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019.png" | prepend: site.baseurl }}">
```

File: data/geoscience/alluvial-fans/index.html [hunk 4]
```diff
@@ -37,6 +39,9 @@ <h3 id="AlluvialFans">
...
+      <img class="inline-text-left outline" src="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019.png" | prepend: site.baseurl }}" alt="Alluvial Fan Mapping Status" />
```
Comment (stdavis) [inline data/geoscience/alluvial-fans/index.html]:
```suggestion
      <img class="inline-text-left outline" src="{{ "/images/Utah_Alluvial_Fan_Inventory_Boundaries_as_of_2019_sm.png" | prepend: site.baseurl }}" alt="Alluvial Fan Mapping Status" />
```

**Comment by rkelson** [comment]
any idea why the deploy previews don't work? they just take me to the WIP page.

**Comment by stdavis** [comment]
Try clicking on this link:
image

I'm not sure what the "This branch has not been deployed" message is about.

**Comment by rkelson** [comment]
Yeah that is what I usually do but it has not been completing lately. Page
looks good. Let's merge and be done with it. Thanks

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
next time there are images in a pr @stdavis, pull it down and compress them before merging

**Comment by rkelson** [comment]
At the request of the data steward can you get the alt="Alluvial Fan
Mapping Status" to give a label/title to the coverage image so it is
similar to this:
[image: image.png]


Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by stdavis** [comment]
We can't see your screenshots when you send them via email...

**Comment by rkelson** [comment]
image

**Pull request merged**